src/main/java/cms/gov/madie/measure/utils/MeasureUtil.java

```diff
@@ -264,4 +266,19 @@ public static List<IncludedLibrary> getIncludedLibraries(String cql) {
                     .build())
         .toList();
   }
+
+  public void validateMetadata(Measure measure) {
```

Comment:
since we are moving this, I think ModelValidator.java would be most appropriate place for this.

Reply:
@adongare That's a good suggestion, just did the change.